src/utils.ts

```diff
@@ -9,7 +9,7 @@ import axios from "axios"
 import { yamlConfig } from "./config"
 import { baseLogger } from "./logger"
 
-export const isProd = process.env.NODE_ENV === "production"
+export const isDev = process.env.NODE_ENV !== "production"
```

Comment:
This burned us on rollout. We should be more narrow in the definition of `isDev`.
```
process.env.NODE_ENV === "development"
```
perhaps?
Don't want this executing on staging.

Reply:
I guess I was using that as `!isProd` but I agree that `isDev` and `isTest` are not ideal. How about we introduce some command options like `--disable-helmet`, `--disable-IP-updates`, `--mock-XYZ` with all of them altering the default behavior of production?